src/itdelatrisu/opsu/translations/LocaleManager.java

```diff
@@ -0,0 +1,608 @@
+/*
+ * opsu! - an open-source osu! client
+ * Copyright (C) 2014-2017 Jeffrey Han
+ *
+ * opsu! is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * opsu! is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with opsu!.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package itdelatrisu.opsu.translations;
+
+import itdelatrisu.opsu.Utils;
+import itdelatrisu.opsu.options.Options.GameOption;
+import itdelatrisu.opsu.util.ListSet;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.IllegalFormatException;
+import java.util.Map;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import java.util.regex.Pattern;
+
+import org.newdawn.slick.util.Log;
+
+import craterstudio.io.Streams;
+
+/**
+ * Provides translation capabilities to opsu!
+ * @author Lyonlancer5
+ */
+public class LocaleManager {
```

Comment:
Something like `Ctrl + Shift + F` in Eclipse?

**Fixed formatting** as per request/code style conformity